### ISSUE TITLE
Full list of Error messages

### ISSUE THREAD

**Issue by SwethaCleo** [jdesgats/ljsonschema#8]
In case there are more than 2 errors in the document, how can we get the complete list? The ljsonschema currently returns only 1 error.

**Comment by jdesgats** [comment]
Yes, I did it that way initially to return as fast as possible as I am using this library to validate untrusted network payloads (actually the error is not even shown in that case).
I guess that trying to carry on might make sense in some cases.
So to answer the question, no, it is currently impossible to get the full list of errors.

**Comment by SwethaCleo** [comment]
Hi @jdesgats , it would be great if that could be done. We can have an flag for the user to either abort on 1st error or capture the full list of errors. Your's is one of the best working jsonschema validators for lua that I have found and if this option is included we need not write custom code in another language only so that we get this added feature of complete errors list. Thanks in advance,

**Comment by jdesgats** [comment]
Hi, I don't have any plans to implement that anytime soon unfortunately. I'd happily review a pull request but I don't have time to work on this at the moment.
